src/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs

```diff
@@ -19,7 +21,8 @@ public static class HostWriter
         /// hash value embedded in default apphost executable in a place where the path to the app binary should be stored.
         /// </summary>
         private const string AppBinaryPathPlaceholder = "c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2";
-        private readonly static byte[] AppBinaryPathPlaceholderSearchValue = Encoding.UTF8.GetBytes(AppBinaryPathPlaceholder);
+        private static readonly byte[] appBinaryPathPlaceholderSearchValue = Encoding.UTF8.GetBytes(AppBinaryPathPlaceholder);
```

Comment:
Naming: 
```suggestion
        private static readonly byte[] AppBinaryPathPlaceholderSearchValue = Encoding.UTF8.GetBytes(AppBinaryPathPlaceholder);
```

Reply:
`private static readonly` typically start with lower case per the .NET Core guild lines.

Reply:
Unfortunately HostModel source code doesn't follow the CoreFx style. I was trying to make it consistent with definitions in some other files. But I'm OK either way.

Reply:
I have capitalize this one back. :)